xml/storage_multipath.xml

```diff
@@ -4617,32 +4617,91 @@ Timed out waiting for device dev-disk-by\x2duuid-c4a...cfef77d.device.
 Welcome to emergency shell
 Give root password for maintenance
 (or press Control-D to continue):</screen>
-   <para>
-    This issue occurs in the following situations:
-   </para>
-   <itemizedlist>
-    <listitem>
-     <para>
-      A non-multipath root file system is not blacklisted when multipath is
-      enabled. See
-      <xref linkend="pro-multipath-trouble-root-blacklist"
-       xrefstyle="HeadingOnPage"/>.
-     </para>
-    </listitem>
-    <listitem>
-     <para>
-      On a system with a multipath root file system, multipath has been
-      enabled/disabled without rebuilding the <filename>initrd</filename>. See
-      <xref linkend="pro-multipath-trouble-root-initrd"/>.
-     </para>
-    </listitem>
-    <listitem>
+    <para>
+     At this stage, you are working in a temporary <command>dracut</command>
+     emergency shell from the initrd environment. To make the configuration
+     changes described below persistent, you need to perform them in the
+     in the environment of the installed system.
+    </para>
+    <important>
      <para>
-      Network-storage drivers are missing in the <filename>initrd</filename> on
-      a system with network-attached storage and multipath enabled.
+      As a general rule, rebuild <filename>initrd</filename> whenever
+      you make changes to <filename>multipath.conf</filename>.
      </para>
-    </listitem>
-   </itemizedlist>
+    </important>
+    <procedure>
+     <step>
+      <para>
+       Identify what the system root (<filename>/</filename>) file system is.
+       For help, see devices under <filename>/dev/disk/by-uuid</filename>,
+       <filename>/dev/disk/by-id</filename>, or
+       <filename>/dev/disk/by-path</filename>.
+      </para>
+     </step>
+     <step>
+      <para>
+       Verify whether the root file system is mounted by using the
+       <command>mount</command> command's output and mount it if needed.
+      </para>
+      <tip>
+       <para>
+        <command>dracut</command> mounts the root file system under
+        <filename>/sysroot</filename> by default.
+       </para>
+      </tip>
+      <para>
+       From now on, let us assume that the root file system is mounted under
+       <filename>/sysroot</filename>.
+      </para>
+     </step>
+     <step>
+      <para>
+       <command>chroot</command> into the <filename>/sysroot</filename>
+       mount point to access the installed system properly. Refer to
+       <xref linkend="sec-trouble-data-recover-rescue-access"/> for more
+       details.
+      </para>
+     </step>
+     <step>
+      <para>
+       Select the right scenario why the system escaped to the emergency shell
```

Comment:
I don't want to over-complicate matters, but this is exactly the problem. **How** would a non-experienced user "select" the right thing here?

Another point, probably to address in a separate / later patch: When the system fails to boot because of multipath issues, it doesn't necessarily happen in dracut's emergency shell. It's actually more likely to happen *after* switching root, when additional file systems or LVM volumes can't be found/activated because of broken multipath configuration.

In either case, the user ends up in systemd's emergency mode. She may have to figure out first if booting failed before or after the root switch. In the latter case, no chrooting is necessary, obviously. Typically you'd check if `/etc/initrd-release` exists, it does so only in the initrd environment (see [systemd documentation](https://systemd.io/INITRD_INTERFACE/)).